src/cljs/rems/actions/change_resources.cljs

```diff
@@ -107,9 +108,10 @@
            :item-key :id
            :item-label #(get-localized-title % language)
            :item-selected? #(contains? (set selected-resources) (% :id))
-           :multi? true
-           :on-change on-set-resources}]]
-        (text :t.actions/bundling-intro)])]))
+           :multi? (:enable-cart config)
+           :on-change #(on-set-resources (flatten (list %)))}]] ; single resource or list
+        (when (:enable-cart config)
+          (text :t.actions/bundling-intro))])]))
```

Comment:
Changing resources is of marginal value if bundling is not used, but that can be disabled separately, I think.